### PR TITLE
[EDU-5526] improve dev command reference

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Small changes on `dev` CLI command guide to clarify the purpose of the `link` command

### CODE REVIEW

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx [hunk 1]
```diff
@@ -19,10 +19,13 @@ In case one or more required fields aren't informed through the specific flags,
 
 ## dev
 
-The `dev` command adapts the environment for building your edge application and run it locally. It's used for projects already initialized as **Azion edge applications**.
+The `dev` command adapts the environment for building your edge application and run it locally. It's used for projects already initialized or linked to an **Azion edge applications**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx:22]:
```suggestion
The `dev` command adapts the environment for building your edge application and run it locally. It's used for projects already initialized or linked to an **Azion edge application**.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx [hunk 2]
```diff
@@ -19,10 +19,13 @@ In case one or more required fields aren't informed through the specific flags,
...
 
-<LinkButton link="/en/documentation/products/cli/link-command/" label="link an existing project to an Azion edge application" severity="secondary" /> 
+:::tip
+If you want to run this command for a project that has not been initialized as an Azion edge application yet, you can run the `link` command it to link the project to Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx:25]:
```suggestion
If you want to run this command for a project that hasn't been initialized as an Azion edge application, you can run the `link` command to link the project to Azion.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx [hunk 3]
```diff
@@ -60,15 +63,8 @@ The `--help` option displays more information about the `dev` command.
 
 The `--firewall` option indicates the function is intended for Edge Firewall.
 
-
 <LinkButton link="/en/documentation/products/guides/edge-functions/first-steps/" label="edge functions for edge applications" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx:66]:
```suggestion
<LinkButton link="/en/documentation/products/guides/edge-functions/first-steps/" label="Edge Functions for Edge Application" severity="secondary" /> 
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx [hunk 4]
```diff
@@ -60,15 +63,8 @@ The `--help` option displays more information about the `dev` command.
 
 The `--firewall` option indicates the function is intended for Edge Firewall.
 
-
 <LinkButton link="/en/documentation/products/guides/edge-functions/first-steps/" label="edge functions for edge applications" severity="secondary" /> 
 
-
 <br />
 
 <LinkButton link="/en/documentation/products/secure/edge-firewall/edge-functions/" label="edge functions for edge firewall" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx:70]:
```suggestion
<LinkButton link="/en/documentation/products/secure/edge-firewall/edge-functions/" label="Edge Functions for Edge Firewall" severity="secondary" /> 
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx [hunk 5]
```diff
@@ -18,13 +18,15 @@ import LinkButton from 'azion-webkit/linkbutton'
 No caso de um ou mais campos obrigatórios não serem informados através das flags específicas, uma mensagem interativa será exibida, solicitando essas informações.
 :::
 
-
 ## dev
 
-O comando `dev` adapta o ambiente para a build de sua edge application e executa o projeto localmente. É usado para projetos já inicializados ou vinculados a uma Azion edge application.
+O comando `dev` adapta o ambiente para a build de sua edge application e executa o projeto localmente. É usado para projetos já inicializados ou vinculados a uma **Azion edge application**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx:23]:
```suggestion
O comando `dev` adapta o ambiente para o build de sua edge application e executa o projeto localmente. É usado para projetos já inicializados ou vinculados a uma **Azion edge application**.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx [hunk 6]
```diff
@@ -18,13 +18,15 @@ import LinkButton from 'azion-webkit/linkbutton'
...
 
-<LinkButton link="/pt-br/documentacao/produtos/azion-cli/link-comando/" label="vincule um projeto em uma Azion edge appliction" severity="secondary" /> 
+:::tip
+Se você deseja executar este comando para um projeto que ainda não foi inicializado como uma Azion edge application, você pode executar o comando `link` para vincular o projeto à Azion.
+:::
 
+<LinkButton link="/pt-br/documentacao/produtos/azion-cli/link-comando/" label="vincule um projeto em uma Azion edge appliction" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx:29]:
```suggestion
<LinkButton link="/pt-br/documentacao/produtos/azion-cli/link-comando/" label="Vincule um projeto em uma Azion edge appliction" severity="secondary" /> 
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx [hunk 7]
```diff
@@ -64,12 +66,6 @@ A opção `--firewall` indica que a função é destinada para o Edge Firewall.
 
 <LinkButton link="/pt-br/documentacao/produtos/guias/edge-functions/primeiros-passos/" label="edge functions para edge applications" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx:67]:
```suggestion
<LinkButton link="/pt-br/documentacao/produtos/guias/edge-functions/primeiros-passos/" label="Edge Functions para Edge Application" severity="secondary" /> 
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx [hunk 8]
```diff
@@ -64,12 +66,6 @@ A opção `--firewall` indica que a função é destinada para o Edge Firewall.
 
 <LinkButton link="/pt-br/documentacao/produtos/guias/edge-functions/primeiros-passos/" label="edge functions para edge applications" severity="secondary" /> 
 
-
 <br />
 
 <LinkButton link="/pt-br/documentacao/produtos/secure/edge-firewall/edge-functions/" label="edge functions para edge firewall" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/dev-command/dev.mdx:71]:
```suggestion
<LinkButton link="/pt-br/documentacao/produtos/secure/edge-firewall/edge-functions/" label="Edge Functions para Edge Firewall" severity="secondary" /> 
```

**Review by MarianaAguilera — APPROVED** [review]
Minor style suggestions

**Review by mrmessa — APPROVED** [review]

**Pull request merged**